tests/zoomus/test_client.py

```diff
@@ -62,6 +62,7 @@ def test_init_creates_all_components(self):
                     "user",
                     "webinar",
                     "group",
+                    "room",
```

Comment:
Since you already made a change here, can I trouble you to alphabetize this list?